transport/rif_comms/node.py

```diff
@@ -303,11 +299,38 @@ def log(self):
         return self._log
 
     def __repr__(self):
-        node = f" node:{pex(self._raiden_service.address)}" if self._raiden_service else ""
+        node = f" RIF Comms Transport node:{pex(self.address)}"
```

Comment:
i have changed this because the raiden_service.address is the hub address always and we use this method for ligth client node representation too

Reply:
coolness :+1: